weapons-on-back/cl_weapons-on-back.lua

```diff
@@ -40,27 +41,34 @@ local SETTINGS = {
     }
 }
 
+--- Other Variables ---
 local attached_weapons = {}
+local me = GetPlayerPed(-1)
+local bone = GetPedBoneIndex(GetPlayerPed(-1), boneNumber)
+local toggle = (1,19) --- Change toggle key here. Currently: left alt.

```

Comment:
Does this syntax of variable initialization actually work?? I've never seen that it Lua before.

Reply:
I'm pretty sure yes, I'm pretty new to lua but I've variables like this before.